.mockery.yaml

```diff
@@ -1,85 +1,82 @@
-# global config
-filename: "{{.InterfaceName}}.go"
-dir: "{{.InterfaceDir}}/mocks"
-outpkg: "mocks"
-mockname: "{{.InterfaceName}}"
-with-expecter: true
-resolve-type-alias: false
-disable-version-string: true
-issue-845-fix: true
-# individual interface config
+dir: '{{.InterfaceDir}}/mocks'
+structname: '{{.InterfaceName}}'
+filename: '{{.InterfaceName}}.go'
+pkgname: mocks
+
+template-data:
+  unroll-variadic: true
```

Comment:
Most of this came straight out of the migrate command output. I did have to add this section, because handling of variadic functions changed.